applications/ray/main.tf

```diff
@@ -109,6 +109,7 @@ module "kuberay-cluster" {
   namespace              = var.ray_namespace
   project_id             = var.project_id
   enable_tpu             = data.google_container_cluster.default.enable_tpu
+  enable_gpu             = var.enable_gpu
```

Comment:
Ray clusters for tests should only use CPUs, the module only supports picking between GPU and TPUs, this is added so we can disable both